modules/trusted-firmware-m/Kconfig.tfm.defconfig

```
@@ -92,6 +92,7 @@ config TFM_PARTITION_CRYPTO
 
 config TFM_ITS_ENCRYPTED
 	bool
+	depends on !SOC_NRF54LM20A_ENGA_CPUAPP
```

```suggestion
	depends on !CRACEN_NEED_MULTIPART_WORKAROUNDS
```
? also makes it more explicit
(BTW `CRACEN_NEED_MULTIPART_WORKAROUNDS` depends on `SOC_NRF54LM20A`, not `SOC_NRF54LM20A_ENGA_CPUAPP`, but it probably is the same or even better)